config/helm/appmesh-controller/test.yaml

```diff
@@ -7,6 +7,8 @@ region: ""
 accountId: ""
 preview: false
 clusterName: ""
+useAwsDualStackEndpoint: 0
+useAwsFIPSEndpoint: 0
```

Comment:
Question - in practice, would it ever make sense to configure these differently than `sidecar.useDualStackEndpoint` and `sidecar.useFipsEndpoint`?